website/docs/reference/resource-configs/tags.md

```diff
@@ -16,59 +16,93 @@ datatype: string | [string]
 
 <File name='dbt_project.yml'>
 
+<VersionBlock lastVersion="1.8">
+
 ```yml
 
-models:
+[models](/reference/model-configs):
   [<resource-path>](/reference/resource-configs/resource-path):
     +tags: <string> | [<string>]
 
-snapshots:
+[snapshots](/reference/snapshot-configs):
   [<resource-path>](/reference/resource-configs/resource-path):
     +tags: <string> | [<string>]
 
-seeds:
+[seeds](/reference/seed-configs):
+  [<resource-path>](/reference/resource-configs/resource-path):
+    +tags: <string> | [<string>]
+
+```
+</VersionBlock>
+
+<VersionBlock firstVersion="1.9">
+
+```yml
+
+[models](/reference/model-configs):
+  [<resource-path>](/reference/resource-configs/resource-path):
+    +tags: <string> | [<string>]
+
+[snapshots](/reference/snapshot-configs):
+  [<resource-path>](/reference/resource-configs/resource-path):
+    +tags: <string> | [<string>]
+
+[seeds](/reference/seed-configs):
+  [<resource-path>](/reference/resource-configs/resource-path):
+    +tags: <string> | [<string>]
+
+[saved-queries:](/docs/build/saved-queries)
   [<resource-path>](/reference/resource-configs/resource-path):
     +tags: <string> | [<string>]
 
 ```
+</VersionBlock>
+
 
 </File>
 </TabItem>
 
 <TabItem value="other-yaml">
 
-<File name='models/resources.yml'>
+<VersionBlock firstVersion="1.9">
 
-```yml
-version: 2
+The following examples show how to add tags to dbt resources in YAML files. Replace `resource_type` with `models`, `snapshots`, `seeds`, or `saved_queries` as appropriate.
+</VersionBlock>
 
-models:
-  - name: model_name
-    config:
-      tags: <string> | [<string>]
+<VersionBlock lastVersion="1.8">
 
+The following examples show how to add tags to dbt resources in YAML files. Replace `resource_type` with `models`, `snapshots`, or `seeds` as appropriate.
+</VersionBlock>
+
+<File name='resource_type/properties.yml'>
+
+```yaml
+resource_type:
+  - name: resource_name
+    config:
+      tags: string
+    # Optional: Add specific properties for models
     columns:
       - name: column_name
-        tags: [<string>]
+        tags: string
         tests:
-          <test-name>:
+          test-name:
             config:
-              tags: <string> | [<string>]
+              tags: string
```

Comment:
I don't know about tags for sub-properties like columns, but since the term is plural, I'd be a little surprised if this change was correct. :/

Reply:
thanks @theyostalservice ! it does work for models (see screenshot). 
i tried seeing if it worked for saved queries, but i keep getting errors. can you tell me the specific command a user can use to run save queries with a specific tag? `dbt sl saved-queries --select tag:santi`, `dbt sl export --sq-name--select tag:santi`, `dbt sl export --select tag:santi`, `dbt sl saved-queries --select tag:santi` didn't work for me?


![Screenshot 2024-12-23 at 12 03 26](https://github.com/user-attachments/assets/4b84d130-a5eb-4cda-b996-f2b44fd45448)